src/svtk/svtk/svfile.py

```diff
@@ -282,10 +282,10 @@ def merge_record_infos(self, new_record, header):
         """
         PROTECTED_INFOS = ('SVTYPE CHR2 END STRANDS SVLEN ALGORITHMS CIPOS '
                            'CIEND RMSSTD MEMBERS').split()
-        records = [r.record for r in self.records]
         infos = defaultdict(list)
 
-        for record in records:
+        for sv_record in self.records:
+            record = sv_record.record
             for info, value in record.info.items():
```

Comment:
I know this predates your changes, but I kind of hate that the key is referred to as `info`, which just creates confusion. I'd suggest renaming it to `key` or `info_key`. (Both here and below)

Reply:
Done, and cleaned up a bit